src/global/global_helpers.R

```diff
@@ -2650,12 +2661,16 @@ calc_inst_flux <- function(chemprod, qprod, site_name){#, dt_round_interv,
         filter(flux_convertible == 1) %>%
         pull(variable_code)
 
-    chem <- read_feather(glue('data/{n}/{d}/munged/{cp}/{s}.feather',
+    chem <- read_feather(glue('data/{n}/{d}/{l}/{cp}/{s}.feather',
                               n = network,
                               d = domain,
+                              l = level,
                               cp = chemprod,
-                              s = site_name)) %>%
-        select(one_of(flux_vars), 'datetime', 'ms_status', 'ms_interp')
+                              s = site_name)) 
+    
+    chem <- chem %>%
+        pivot_wider(names_from = 'var', values_from = 'val') %>%
+        select(contains(flux_vars), 'datetime', 'ms_status', 'ms_interp')
```

Comment:
Why `contains` instead of `one_of`? Wouldn't `contains` allow unforeseen selections if a non-flux variable contains the same string as a flux variable?